docs/Third-party.md

```diff
@@ -1,281 +1,268 @@
-| `t1-freesurfer`          |      |           |     x      |     |     |        |         |        |     |
-| `dwi-preprocessing-*`    |   x  |     x     |            |  x  |     |        |    x    |        |     |
-| `dwi-dti`                |   x  |           |            |  x  |     |        |    x    |        |     |
-| `dwi-connectome`         |   x  |           |     x      |  x  |     |        |    x    |        |     |
-| `pet-surface`            |      |           |            |  x  |  x* |        |         |   x*   |  x  |
-| `pet-volume`             |      |           |            |     |  x* |    x   |         |   x*   |  x  |
-| `statistics-surface`     |      |           |            |     |     |    x   |         |        |     |
-| `machine-learning-*`     |      |           |            |     |     |        |         |        |     |
-
-!!! note "CAT12 toolbox"
-    Starting from Clinica `v0.3.7`, the [**CAT12**](http://dbm.neuro.uni-jena.de/cat/) toolbox is no longer needed for the `t1-volume` and `pet-volume` pipelines.
-    For previous versions of Clinica, you will need to download the latest version of the toolbox [here](http://dbm.neuro.uni-jena.de/cat/index.html#DOWNLOAD) and follow the instructions to ensure that your `cat12` folder is located in your `spm/toolbox` folder.
+!!! danger "Provided command lines should be adapted"
+    **Please** do not copy/paste the provided commands without adapting them to your system and without testing them.
+    Most paths provided here require to be adapted to your system and depend on how you installed the software.
+    
+    A good approach is to verify that the different paths you want to assign to a variable exist first : 
 
-_*You only need to install ITK if you plan to perform partial volume correction using PETPVC._
+    :material-arrow-right: If they do, then try running the `export` and `source` **(1.)** commands in your terminal. Verify that the software or the pipeline you want runs as expected.
+    
+    :material-arrow-right: If this works, then consider modifying your shell configuration file **(2.)** to have these variables automatically defined on every session.
 
-Depending on the architecture and OS of your system, setup of third party libraries can change.
-Please refer to each tool’s website for installation instructions:
 
-### Environment variables setup
+## Converters
 
-When installing some of the third party software, environment variables might be needed in order to configure some installations.
+Some converters require **dcm2niix** to transform DICOM files into NIfTI :
 
-If you simply run the `export` and `source` commands in your terminal, these environment variables will be defined only for the duration of your session, and will be lost if you re-launch your terminal.
+<div class="grid">
+  <a href="Converters/ADNI2BIDS.md" class="card">adni-to-bids</a>
```

Comment:
These links aren't working. I think you need to provide the path to the html page, not the markdown file.

Reply:
You're right. I am now using https://aramislab.paris.inria.fr/clinica/docs/public/dev/ (+ page) as it is described in the mkdocs.yml as the site_url, WDYT ?

Reply:
Hmmm, this is the URL of the development version of the documentation. This will cause some issues because old versions of the doc will, at some point in the future, point to dev versions for example. Also, hardcoding the URL in the code isn't great. If you need to change it, you have to change it all over the code base.
I think you should use a relative path to the html file, not the URL of a deployed version of the doc. Does this make sense ?